api/swagger-spec/oapi-v1.json

```diff
@@ -22067,7 +22071,7 @@
      },
      "config": {
       "$ref": "v1.ObjectReference",
-      "description": "config is an ObjectReference to the BuildConfig this Build is based on."
+      "description": "config is an ObjectReference to the BuildConfig this Build is based on. This field is deprecated in deference to Spec.Config."
```

Comment:
the conversion still respects status if it's provided, though....  spec is only used if it's provided and non-nil, isn't that sufficient to be non-breaking for older clients?